src/SmartComponents/CompletedTaskDetails/JobResultsDetails/ExpandedIssues.js

```diff
@@ -73,7 +73,11 @@ const ExpandedIssues = ({ rowPairs, isReportJson }) => {
             backgroundColor: '#ffffff',
           }}
         >
-          <CodeBlockCode>{rowPairs}</CodeBlockCode>
+          <CodeBlockCode>
+            {rowPairs instanceof Object
```

Comment:
instanceof Object is not recommended by the JS community as it fails to properly check the type. Reference: https://bobbyhadz.com/blog/javascript-check-if-value-is-object#dont-use-the-instanceof-operator-to-check-if-a-value-is-an-object. Can we use another assertion?

Reply:
Thanks for the feedback @mkholjuraev.  How about the typeof operator?